packages/changed-elements-react/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased](https://github.com/iTwin/changed-elements-react/tree/HEAD/packages/changed-elements-react)
 
+### Patch changes
+
+* Added props for disable side by side toggle in PropertyComparisonTable
```

Comment:
Use present tense, be more concrete so that a person knows how to use the new feature.

```suggestion
* `PropertyComparisonTable`: Add ability to hide side-by-side comparison toggle by setting `displaySideBySideToggle` property to `false`
```